src/nplinker/genomics/antismash/antismash_downloader.py

```diff
@@ -0,0 +1,420 @@
+import csv
+import os
+import re
+import time
+import zipfile
+import httpx
+from bs4 import BeautifulSoup
+from progress.bar import Bar
+from nplinker.logconfig import LogConfig
+
+
+logger = LogConfig.getLogger(__name__)
+
+
+ANTISMASH_DB_PAGE_URL = 'https://antismash-db.secondarymetabolites.org/output/{}/'
+ANTISMASH_DB_DOWNLOAD_URL = 'https://antismash-db.secondarymetabolites.org/output/{}/{}'
+
+ANTISMASH_DBV2_PAGE_URL = 'https://antismash-dbv2.secondarymetabolites.org/output/{}/'
+ANTISMASH_DBV2_DOWNLOAD_URL = 'https://antismash-dbv2.secondarymetabolites.org/output/{}/{}'
```

Comment:
The `dbv2` database is not official any more. We need to remove it and update related code.